images/run/Dockerfile

```diff
@@ -9,6 +10,8 @@ HEALTHCHECK --interval=30s --timeout=1s --start-period=60s --retries=1 \
 
 USER pack
 
+LABEL io.buildpacks.stack.id="${stack_id}"
+ENV PACK_STACK_ID="${stack_id}"
```

Comment:
Should not be exposed for launch.

Reply:
Makes sense for `PACK_STACK_ID` but the label is useful because the platform could use it for validation. For example, when `pack build` is given a `--run-image` flag, we need to check the stack.